packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -446,6 +446,11 @@ interface CommonRotationUserOptions {
  * Options to add the multi user rotation
  */
 export interface RotationSingleUserOptions extends CommonRotationUserOptions {
+  /**
+   * The secret to rotate, must be already attached!
```

Comment:
1. There should be an empty line after this line:

```suggestion
   * The secret to rotate, must be already attached!
   *
```

2. We should call `attach` ourselves, we shouldn't require the customer to do that.

Reply:
This is not the same behavior CDK has in multi-user rotation (it expects the secret to be attached already). Related to #13719. What do you think?